core/lib/generators/solidus/paperclip_adapter/install/install_generator.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails/generators'
+
+module Solidus
+  # @private
+  module PaperclipAdapter
+    class InstallGenerator < Rails::Generators::Base
+      source_root File.expand_path('templates', __dir__)
+
+      class_option :app_directory,
+        type: :string,
+        default: Rails.root,
+        hide: true
+
+      class_option :initializer_directory,
+        type: :string,
+        default: 'config/initializers/',
+        hide: true
+
+      class_option :initializer_basename,
+        type: :string,
+        default: 'spree',
+        banner: 'The name of the initializer containing the attachment adapters settings'
+
+      class_option :app_name,
+        type: :string,
+        default: 'MyStore',
```

Comment:
I think we can automate that with `Rails.application.class.module_parent.name`

Reply:
It was on my metal TODO list, and of course, I forgot about it. Thanks!

Reply:
> metal TODO list

:metal:

Reply:
lol 🤘